.github/workflows/main.yml

```diff
@@ -136,24 +142,36 @@ jobs:
         run: sudo dpkg -i /var/cache/apt/archives/*.deb
 
       - name: Install Lua packages
+        if: matrix.lua_version != '5.4'
         run: |
           if [ "${{ matrix.lua_name }}" = "luajit" ]; then
             sudo apt-get install libluajit-5.1-dev luajit
           fi
           sudo apt-get install liblua${{ matrix.lua_version }}-dev lua${{ matrix.lua_version }}
 
+      # Ubuntu 20.04 hasn't a package for Lua 5.4, we need to build it from source.
+      - name: Build and Install Lua 5.4
+        if: matrix.lua_version == '5.4'
+        run: |
+          wget -O /tmp/lua-5.4.3.tar.gz http://www.lua.org/ftp/lua-5.4.3.tar.gz
+          tar -xf /tmp/lua-5.4.3.tar.gz -C /tmp
+          cd /tmp/lua-5.4.3
+          make all
+          make test
```

Comment:
The Lua doc recommends running `make test` when building to "be sure there is no error". I don't know if we want to keep that, tho...

Reply:
mb depends how much time does it take

Reply:
On my computer, I'd says `make test` is nearly instantaneous. On the GitHub Actions, the whole step "Build and Install Lua 5.4" is about 10 seconds.

So I guess we can keep this `make test` instruction :shrug: